safe-global/safe-homepage
# fix: Add CLA page

## What it solves

- Adds missing CLA page under `/cla`

## How to test

1. Open the page
2. Navigate to `/cla`
3. Observe the same content as in https://github.com/safe-global/safe-landing-page/blob/dev/src/pages/cla.tsx

## Screenshot

<img width="1478" alt="Screenshot 2023-04-05 at 15 54 48" src="https://user-images.githubusercontent.com/5880855/230102344-ac64c104-6c49-4b02-836a-f1e36afaf573.png">
